src/components/dashboard/DashboardKeys.js

```diff
@@ -70,11 +69,13 @@ const DashboardKeys = ({ image, title, to, accessKeys, empty }) => (
         <Grid.Row>
             <Grid.Column textAlign='left' width={16}>
                 {accessKeys && accessKeys.length !== 0 ? (
-                    <Link to={to}>
-                        <FormButton color='gray-blue' size='small'>
-                            <Translate id='button.viewAll' />
-                        </FormButton>
-                    </Link>
+                    <FormButton
+                        to={to}
```

Comment:
Would it make sense to name URL prop something else? `to` doesn't help me understand what it is outside of FormButton context.

Reply:
`linkTo`?